credentialproviderpackage/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/aws/eks-anywhere-packages/credentialproviderpackage
 go 1.19
 
 require (
+	github.com/Masterminds/semver v1.5.0
```

Comment:
Can you use `"golang.org/x/mod/semver"`
We use it here: https://github.com/aws/eks-anywhere-packages/blob/main/pkg/bundle/manager.go#L10